dojo/models.py

```diff
@@ -1229,9 +1229,9 @@ class Finding(models.Model):
     title = models.TextField(max_length=1000)
     date = models.DateField(default=get_current_date)
     cwe = models.IntegerField(default=0, null=True, blank=True)
-    cve_regex = RegexValidator(regex=r'^CVE-\d{4}-\d{4,7}$',
-                                 message="CVE must be entered in the format: 'CVE-9999-9999'. ")
-    cve = models.CharField(validators=[cve_regex], max_length=20, null=True)
+    cve_regex = RegexValidator(regex=r'^[A-Z]{1,10}-\d{4}-\d{4,12}$',
+                                 message="Vulnerability ID must be entered in the format: 'ABC-9999-9999'. ")
+    cve = models.TextField(validators=[cve_regex], max_length=28, null=True)
```

Comment:
The db_migration file gets created automatically from the models.py. This is why your Travis is failing.

Reply:
Thx, fix this